packages/flutter/lib/src/material/slider.dart

```diff
@@ -287,6 +289,16 @@ class Slider extends StatefulWidget {
   /// appearance of various components of the slider.
   final Color inactiveColor;
 
+  /// Whether to normalize the semantic value of the slider to a percentage.
+  ///
+  /// This is used by accessibility frameworks like TalkBack on Android to
+  /// inform users what the currently selected value is. If false, the value
+  /// is communicated instead of the percentage rounded to the nearest whole
+  /// number.
+  ///
+  /// Defaults to true.
+  final bool normalizedSemanticsValue;
```

Comment:
we should make this an enum, so we can extend it later.

Reply:
From offline discussion, we're going to make this take a formatting callback